assessments/projects/in-toto/self-assesment.md

```diff
@@ -0,0 +1,783 @@
+﻿# in-toto security self-assessment 
+## with input from the SIG-Security WG
+
+June 5, 2019
+
+*_Authors_*

```

Comment:
line breaks below don't appear in rendered markdown, so I think it would be a little more readable if it was 

*_Authors_*: name, name, name 

keeping markdown source as separate lines for easier reading/diffing of source